src/containers/AccountConnection.jsx

```diff
@@ -37,6 +37,10 @@ class AccountConnection extends Component {
     })
   }
 
+  componentDidMount () {
+    document.querySelector('.coz-giveFocus').focus()
```

Comment:
You should not use the `document.querySelector` here, you can use `ref` instead directly in the `Field` JSX component. See the docs about refs here: https://facebook.github.io/react/docs/refs-and-the-dom.html#adding-a-ref-to-a-dom-element
A potential issue here can be that you'll have to have a functional component to use them but I am not sure I let you figure it out.